addons/danger/functions/fnc_shareInformation.sqf

```diff
@@ -35,25 +35,32 @@ _unit setVariable [QGVAR(currentTask), "Share Information", GVAR(debug_functions
 
 // find units
 private _groups = allGroups select {
-    leader _x distance2d _unit < _range
+    leader _x distance2D _unit < _range
     && {[side _x, side _unit] call BIS_fnc_sideIsFriendly}
     && {behaviour leader _x != "CARELESS"}
     && {_x != group _unit}
 };
 
 private _knowsAbout = _unit knowsAbout _target;
 
-
 // share information
 {
+    // share information
     if !(isNull _target) then {
         [_x, [_target, _knowsAbout min GVAR(maxRevealValue)]] remoteExec ["reveal", leader _x];
+
+        // reinforce
+        if (_range > 100 && {_x getVariable [QGVAR(enableGroupReinforce), false]}) then {
+            [_x, [getpos _unit, (_unit targetKnowledge _target) select 6] select (_knowsAbout > 0.5)] remoteExec ["move", leader _x];
+        };
     };
 
-    if ((leader _x) distance _unit < (200 min _range)) then {
-        _x setBehaviour "COMBAT";
-        _x setFormDir ((leader _x) getDir _unit);
+    // set combat mode
+    if ((leader _x) distance2D _unit < ((GVAR(combatShareRange)) min _range) && {!((leader _x) getVariable [QGVAR(disableAI), false])}) then {
+        [_x, "COMBAT"] remoteExec ["setBehaviour", leader _x];
```

Comment:
didn't we had an issue with remoteExec on Units and used Groups?

Reply:
should be fine when run on "leader _x"